pom.xml

```diff
@@ -15,7 +15,7 @@
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Promoted+Builds+Plugin</url>
 
   <properties>
-    <jenkins.version>1.565</jenkins.version>
+    <jenkins.version>1.614</jenkins.version>
```

Comment:
Noted, will be addressed, 1.614 was chosen as the oldest version being build-compatible with `job-dsl-plugin` 1.46 (which introduced the Automatically Generated DSL)